freezeyt/freezer.py

```diff
@@ -197,12 +233,20 @@ def __init__(self, app, config):
             ('extra_pages', ()),
             ('extra_files', None),
             ('default_mimetype', 'application/octet-stream'),
-            ('get_mimetype', default_get_mimetype),
+            ('get_mimetype', default_mimetype),
+            ('mime_db_file', None),
             ('url_to_path', default_url_to_path)
         )
         for attr_name, default in CONFIG_DATA:
             setattr(self, attr_name, config.get(attr_name, default))
 
+        if self.mime_db_file:
+            with open(self.mime_db_file) as file:
+                mime_db = json.load(file)
+
+            mime_db = mime_db_conversion(mime_db)
```

Comment:
I am not sure about same name, because content is same but structure is different